test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments proceed as following:
+
+### Generate the workload traffic
+
+1. Clone the following clusterloader2 repo: 
+
+   ```bash
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory
+
+   ```bash
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   
+   ```bash
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numbeOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
+  
+   To learn more about clusterloader2 tuningSet, see the following: [tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```bash
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```bash
```

Comment:
FYI, for displaying output rather than bash commands, you might want to use the language tag `console` rather than `bash`. Compare:

```bash
Switched to context "wds1".
```

```console
Switched to context "wds1".
```

Reply:
Fixed - using `console` instead of `bash` now

Reply:
I was trying to suggest using the language tag "console" for displayed output, not displayed commands. For displayed commands the language tag "bash" is a more accurate description. And having the two sorts of text rendered in different colors will be helpful to the reader.